README.sdk.md

```diff
@@ -121,6 +121,16 @@ for data in subscribed_data:
 
 The Nyx ecosystem is at an early stage of its development, please give us feedback through [Github issues](https://github.com/Iotic-Labs/nyx-sdk/issues).
 
+- If you get the SSL error 
+`httpcore.ConnectError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate`
+
+then it may be that you need to set some environment variables like this before running your script:
+```shell
+CERT_PATH=$(python -m certifi)
+export SSL_CERT_FILE=${CERT_PATH}
+export REQUESTS_CA_BUNDLE=${CERT_PATH}
+```
```

Comment:
If problem persists add this to your .bashrc/.zshrc ?

Reply:
I think leave that to them to decide?